Lib/imaplib.py

```diff
@@ -1279,6 +1385,236 @@ def print_log(self):
                 n -= 1
 
 
+class _Idler:
+    # Iterable context manager: start IDLE & produce untagged responses
+    #
+    # This iterator produces (type, datum) tuples.  They slightly differ
+    # from the tuples returned by IMAP4.response():  The second item in the
+    # tuple is a single datum, rather than a list of them, because only one
+    # untagged response is produced at a time.
```

Comment:
Should be in a string:
```suggestion
    """Iterable context manager: start IDLE & produce untagged responses
    
    This iterator produces (type, datum) tuples.  They slightly differ
    from the tuples returned by IMAP4.response():  The second item in the
    tuple is a single datum, rather than a list of them, because only one
    untagged response is produced at a time."""
```

Reply:
I think we do need to document it (see my other comment). It's going to have to go into typeshed anyway, we can't easily just pretend it doesn't exist in the future.